### ISSUE TITLE
x2go

### ISSUE THREAD

**Issue by danboid** [haikuports/haikuports#6499]
It'd be great to see the x2go client ported to Haiku. x2go is my preferred way to remote into X. It gives smoother video than VNC and also handles sound. It has Qt and python based clients

https://wiki.x2go.org/doku.php/doc:installation:start

**Comment by extrowerk** [comment]
![screenshot226](https://user-images.githubusercontent.com/5569059/158547161-fb800c17-9fd2-4ba7-b939-41b476bf8231.png)
Had to remove the print support, LDAP is also disabled but otherwise it should work. No way to test it tho.

**Comment by danboid** [comment]
Very cool! I shall test it ASAP but I'm not sure on what because my Haiku box died yesterday. I've been thinking about buying a new PC recently and I've got a good reason now although I think I'll be lucky to get hardware that can boot Haiku.

You don't have a RPi or a Linux box? Installing x2go server is very easy under most distros and there is often no config required.

**Comment by extrowerk** [comment]
> Very cool! I shall test it ASAP but I'm not sure on what because my Haiku box died yesterday. I've been thinking about buying a new PC recently and I've got a good reason now although I think I'll be lucky to get hardware that can boot Haiku.

I have yet to wri te a recipe.

> You don't have a RPi or a Linux box? Installing x2go server is very easy under most distros and there is often no config required.

Sorry, but i am not a fan of nixes.

**Comment by waddlesplash** [comment]
I don't think it will work because it generally requires a "real" X11 server to forward to...

**Comment by danboid** [comment]
@waddlesplash I don't think that the Window and Mac x2go clients install xservers and I don't see why they would need to. VNC doesn't require non X users install an X server either.

**Comment by waddlesplash** [comment]
Windows X2Go comes with its own X server.
